swo_playground/api.py

```
@@ -44,40 +44,40 @@ def jwt_secret_callback(client: MPTClient, claims: Mapping[str, Any]) -> str:
 
-@ext.api.get(
+@ext.api.get(  # type: ignore[untyped-decorator]
     "/",
     description="Root endpoint.",
     response={
         200: dict,
         400: Error,
     },
-)  # type: ignore[misc]
+)
 def root(request: ExtensionHttpRequest) -> Response:
```

Ruff (0.14.13)

Unused function argument: `request`

(ARG001)